tests/sentry/templatetags/test_sentry_assets.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from django.template import Context, Template
+from mock import Mock
+
+from sentry.testutils import TestCase
+
+
+class FeaturesTest(TestCase):
+    TEMPLATE = Template("""
+        {% load sentry_assets %}
+        {% locale_js_include request %}
+    """)
+
+    def test_supported_foreign_lang(self):
+        result = self.TEMPLATE.render(Context({
+            'request': Mock(LANGUAGE_CODE='fr'),  # French, in locale/catalogs.json
+        }))
+
+        assert '<script src="/_static/{version}/sentry/dist/moment/locale/fr.js" ></script>' in result
+
+    def test_unsupported_foreign_lang(self):
+        result = self.TEMPLATE.render(Context({
+            'request': Mock(LANGUAGE_CODE='ro'),  # Romanian, not in locale/catalogs.json
```

Comment:
I'd prefer this test some garbage string. Like, `LANGUAGE_CODE='this-will-never-happen'`.

If we introduce Romanian support one day, the test will break.